app/views/shared/nav/_nav_links.html.haml

```diff
@@ -1,4 +1,8 @@
 %ul.nav
+  %li{class: [home_button.tab_class(controller), "useButton"], id: home_button.tab_id }  
```

Comment:
Looks like in the merge conflicts it's still in the `responsive?` section. I'll clean that out as part of the merge.